velox/common/caching/SsdFile.cpp

```diff
@@ -478,6 +483,7 @@ bool SsdFile::write(
       << ", size: " << iovecs.size() << ", offset: " << offset
       << ", error code: " << errno
       << ", error string: " << folly::errnoStr(errno);
+  RECORD_METRIC_VALUE(kMetricSsdCacheWriteSsdErrors)
```

Comment:
```
s/kMetricSsdCacheWriteSsdErrors/kMetricSsdCacheWriteErrors/
```

Reply:
Discussed with @tanjialiang during the recent refactor. Better keep them compatible for now. Changing the naming convention will break the dashboards. If refactor is indeed needed, better have a separate PR to make the change as this is not the only metric having this naming convention.